plugins/basic/info.yml

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+
+# This file contains the plug-in configuration data that will be packaged
+# into manifest.py by the build_plugin.py script.
+
+
+#
+# Plug-in information recommended values to specify by convention.
+#
+
+# Plug-in name that will prefix the plug-in messaqges displayed in Maya script editor.
+name: "tk-maya-basic"
+
+# Plug-in version number that will be displayed in Maya Plug-in Information window.
+# This plug-in version number and the one in the Maya module file should be synchronised.
+version: "0.0.5"
+
+description:  "Shotgun toolkit integration into a basic plug-in."
```

Comment:
Hmm, if description isn't used by the maya plugin in any way we should drop it. These are optional and implementation specific parameters. Does that make sense?